certora/specs/AllowancesInvariant.spec

```
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+methods {
+    function _.approve(address, uint256 amount)  external => summaryApprove(amount) expect bool;
```

```suggestion
    function _.approve(address account, uint256 amount) external => summaryApprove(amount) expect bool;
```
then change the summary in this:

```    
function summaryApprove(address account, uint256 amount) returns bool {
     if account is not in known immutable contracts {
         unsafeApproveCalled = true;
         lastUnsafeApproveNull = amount == 0;   
     }
     bool res;
     return res;
 }

---

Done in https://github.com/morpho-org/bundler3/pull/238/commits/b2d7065f26ba6cb1ccc238294fd56b3307bbd36e.